plugin/pkg/scheduler/api/validation/validation_test.go

```diff
@@ -76,8 +76,8 @@ func TestValidateExtenderWithNegativeWeight(t *testing.T) {
 func TestValidateMultipleExtendersWithBind(t *testing.T) {
 	extenderPolicy := api.Policy{
 		ExtenderConfigs: []api.ExtenderConfig{
-			{URLPrefix: "http://127.0.0.1:8081/extender", BindVerb: "bind"},
-			{URLPrefix: "http://127.0.0.1:8082/extender", BindVerb: "bind"},
+			{URLPrefix: "http://127.0.0.1:8081/extender", BindVerb: "bind", Weight: 2},
+			{URLPrefix: "http://127.0.0.1:8082/extender", BindVerb: "bind", Weight: 2},
 		},
 	}
 	if ValidatePolicy(extenderPolicy) == nil {
```

Comment:
I think we need to check error message to make sure `ValidatePolicy` reports multiple binder error.